### DISCUSSION TITLE
一个单核cpu安装问题

### DISCUSSION THREAD

**Discussion by VincentSong524** [coder/code-server#5111] [Q&A]
一核CPU是否能够安装并运行这个项目

**Comment by jsjoeio** [comment]
Hmm...can you translate to English?

**Reply by Qubik65536** [reply]
He is trying to say

> Is a single core CPU enough for install and run this program.

**Reply by jsjoeio** [reply]
Ah thank you! 

It _might_ be. I would test it and see!

**Reply by Qubik65536** [reply]
> Ah thank you! 
> 
> 
> 
> It _might_ be. I would test it and see!

I tried it on my own server, it's actually fluent as long as I just use it as an online editor with just a few plugins even though it's not a good CPU. I think it depends on what kind of work it's doing because running the code needs indeed extra performance.

**Reply by jsjoeio** [reply]
Ah okay. Yeah, I think you're right. Basic stuff works but anything that requires a good amount of CPU, it might not work well.
